src/HasPrefixCollection.php

```
@@ -57,16 +57,18 @@ public function getPrefix(): string
      *
      * @param string|UnitEnum $key Attribute name.
      * @param mixed $default Default value when the attribute is missing.
+     * @param string $prefix Prefix to ensure (for example, `aria-`, `data-`, `on`).
      *
      * @return mixed Attribute value or default.
      */
-    public function getPrefixAttribute(string|UnitEnum $key, mixed $default = null): mixed
+    public function getPrefixAttribute(string|UnitEnum $key, mixed $default = null, string $prefix = ''): mixed
     {
-        return AttributeBag::get($this->prefixAttributes, $key, $default);
+        return AttributeBag::get($this->prefixAttributes, $key, $default, $prefix);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Helper\AttributeBag' in method 'getPrefixAttribute'. (undefined)

(StaticAccess)